packages/custom-esbuild/src/dev-server/index.ts

```diff
@@ -42,7 +39,7 @@ export function executeCustomDevServerBuilder(
       const middleware = await Promise.all(
         (options.middlewares || []).map(middlewarePath =>
           // https://github.com/angular/angular-cli/pull/26212/files#diff-a99020cbdb97d20b2bc686bcb64b31942107d56db06fd880171b0a86f7859e6eR52
-          loadModule<Connect.NextHandleFunction>(
+          loadModule<any>(
```

Comment:
You can take the following file from my PR:

https://github.com/just-jeb/angular-builders/pull/1871/files#diff-5768dc9114dc59a71cc5955282b7b5e1a3e4028ea49a9a753cb66477bb7e59fa